FinSearchUnified/Commands/ExportCommand.php

```diff
@@ -25,13 +26,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
         /** @var ShopwareProcess $blController */
         $blController = $this->container->get('fin_search_unified.shopware_process');
         $shopkey = $input->getArgument('shopkey');
+        $language = 'de_DE';
+
+        if ($input->getArgument('language') !== null) {
```

Comment:
Usually when you work with query parameters, you can provide a default value if the parameter doesn't exist. Is sth. similar possible here?
```
$language = $input->getArgument('language', 'de_DE');
```
Just would look nicer.

Reply:
@ihanli :+1: 
the `addArgument` method supports a default argument. Thanks for that!